openstates/ok/bills.py

```diff
@@ -155,11 +155,12 @@ def scrape_votes(self, bill, url):
                 motion_index = 8
             else:
                 chamber = 'upper'
-                motion_index = 9
+                motion_index = 13
```

Comment:
@mileswwatkins can you comment this? It's weird and takes some doing to figure out